start_group.html

```diff
@@ -0,0 +1,888 @@
+                          </div>
+                        </div>
+                        <div class="text info aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                          <div id="text-924f8b967f" class="cmp-text">
+                              <p>Discuss highly recommended leadership resources by world-class leaders.</p>
+                          </div>
+                        </div>
+                      
+                    </div>
+                  </div>
+                  <div class="container responsivegrid slides aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                    <div id="container-112257af9a" class="cmp-container">
+                        <div class="s-no heading aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                          <div id="title-63091b8133" class="cmp-title">
+                            <h6 class="cmp-title__text">04</h6>
+                          </div>
+                        </div>
+                        <div class="title aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                          <div id="title-63091b8133" class="cmp-title">
+                            <h3 class="cmp-title__text">Apply</h3>
+                          </div>
+                        </div>
+                        <div class="text info aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                          <div id="text-924f8b967f" class="cmp-text">
+                              <p>Apply valuable skills and increase your impact.</p>
+                          </div>
+                        </div>
+                      
+                    </div>
+                  </div>
+                
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+      <!-- START GROUP IMPACT -->
+      <div class="container responsivegrid impact-story aem-GridColumn aem-GridColumn--default--12">
+        <div id="impact-container" class="cmp-container">
+          <div class="aem-Grid aem-Grid--12 aem-Grid--default--12 ">
+            <div class="container responsivegrid stories-content aem-GridColumn aem-GridColumn--default--4 layout-grid">
+              <div id="stories-content-wrap" class="cmp-container">
+                <div class="title aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                  <div id="title-63091b8133" class="cmp-title">
+                    <h2 class="cmp-title__text">Impact Stories</h2>
+                  </div>
+                </div>
+                <div class="container responsivegrid aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                  <div id="stories-item-wrap" class="cmp-container">
+                    <div class="teaser story-item aem-GridColumn aem-GridColumn--default--12 layout-grid" data-val="#tabs-31c70c9208-item-b17065cfe5-tabpanel">
+                      <div class="cmp-teaser">
+                        <div class="cmp-teaser__image">
```

Comment:
@Prystal254 The image/content order needs correcting and the anchor needs to be in the right place.